lib/superset/dashboard/import.rb

```diff
@@ -46,7 +44,9 @@ def response
       def validate_params
         raise ArgumentError, 'source_zip_file is required' if source_zip_file.nil?
         raise ArgumentError, 'source_zip_file does not exist' unless File.exist?(source_zip_file)
+        raise ArgumentError, 'source_zip_file is not a zip file' unless File.extname(source_zip_file) == '.zip'
         raise ArgumentError, 'overwrite must be a boolean' unless [true, false].include?(overwrite)
+        raise ArgumentError, "zip target database does not exist: #{zip_database_config_not_found_in_superset}" if zip_database_config_not_found_in_superset.present?
```

Comment:
This will confirm that the provided import zip file contains a database yaml that is a valid database connection for the target superset environment.

ie .. if we try to just import a Staging dashboard to Production ... it will fail as the staging db connection does not exist on production superset